packages/legend-art/src/components/panel/PanelListSelectorItem.tsx

```diff
@@ -17,11 +17,13 @@ import { clsx } from 'clsx';
 
 export const PanelListSelectorItem: React.FC<{
   title?: string;
+  children?: React.ReactNode;
```

Comment:
When you design a component, it's weird to do thing like `title? children?` we need to handle the logic where both is provided, which one will take precedence. I think you misunderstood my comment earlier as well, see my comment about usage of `{children}` below

Reply:
A neat way is to create another component

`PanelListSelectorItemLabel` which takes just the title and can be used as child of `<PanelListSelectorItem>` which takes a `children` prop only.

Reply:
true!